src/main.rs

```diff
@@ -40,6 +45,20 @@ fn main() {
     // we can have an infinite loop that will read these
     // for files.
     loop {
+
+        if(args.len() > 1) {
+          if(args[1] == "-t") {
+            let start2 = SystemTime::now();
+            let since_the_epoch2 = start2
+                .duration_since(UNIX_EPOCH)
+                .expect("Time went backwards"); 
```

Comment:
Same comment goes for this, give a more explicit error message - something like "Error in calculating time since second epoch occurred" is much better.